dbschema/models.py

```diff
@@ -48,6 +49,17 @@ class DialogQuestions(Base):
     __tablename__ = 'dialog_questions'
     question_id = Column(Integer, primary_key=True)
     question_description = Column(String)
+    
+    
+class FirstAidKit(Base):
+    __tablename__ = 'first_aid_kit'
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    users_nicedayuid = Column(Integer, ForeignKey('users.nicedayuid'))
+    activity_description = Column(String)
```

Comment:
Instead of having one field for both user-defined activites and activities that are in our database, I suggest to have 2 fields. `activity_id`, that refers to 'our' activity, and `user_activity_description`. And maybe document that either one of these should be filled. This will make it easier to link to the correct activity in our database, but still allows the user to come up with their own.